src/MMM01.md

```diff
@@ -0,0 +1,190 @@
+# MMM01
+
+The MMM01 is a mapper specific to multi-game compilation cartridges. It emulates an MBC1 for the contained games, and supports containing a mix of games from 32 kiB ROM with no RAM up to the same maximum memory _per game_ as the MBC1:
+
+* max 512 kiByte ROM with banked 8, 16, or 32 kiByte RAM (default configuration)
+* max 2 MiByte ROM with unbanked 8 kiByte RAM ("multiplex" mode[^multiplex])
+
+[^multiplex]:
+However, no cartridges were released using the MMM01 "multiplex" mode.
+
+Regardless of the size or number of the games in the compilation, the maximum total cartridge size supported by the MMM01 is the same - up to 8 MiB ROM and 128 kiB RAM.
+
+Additional bits in the control registers extend the MBC1 ROM and RAM banking numbers to allow for game selection, and the MBC1 ROM/RAM banking registers can be masked so some of _those_ bits can also be used for game selection, allowing for up to 255x 32 kiB games, plus a 32kiB menu, in an 8 MiB ROM. RAM is more limited at only up to 16x 8kB RAM banks in total. However, despite these generous capabilities, no MMM01 cartridge was released with more than 4 games, and only _one_ contains any RAM at all.
+
+The ROM and RAM "game select" bank numbers do not have to be set to the same number - this allows an MMM01 cartridge to not waste RAM space on games that do not have RAM[^no_ram], or mix and match games that have different sized ROM or RAM by packing them in tightly in the overall ROM/RAM of the cartridge.
+
+[^no_ram]:
+Technically, the MMM01 can't completely block access to RAM for a game, so if the cartridge contains RAM it's recommended to assign any no-RAM games to the same single RAM bank to prevent no-RAM games from accessing or corrupting other games' SRAM saves.
+
+MMM01 starts up in a unique mode ("unmapped") which always maps the _last 32 kiB_ of the ROM to the 0000-7FFF address region regardless of the values in the bank/mode registers. The correct ROM header (with Nintendo logo) therefore needs to be located at offset `(size - 32 kiB) + 100h` in the ROM rather than the usual `0000 + 100h` (which contains the header of the first game in the collection instead). MMM01 cartridges have a simple menu program in this last 32 kiB, which manipulates the additional MMM01 control bits, allowing game selection and setting the game size, before entering "mapped" mode and booting the selected game (see "Mapping Enable" below).
+
+As the last 32 kiB of the ROM are reserved for the cartridge menu, it's best to pack games into the ROM from largest to smallest to avoid having a game overlap the menu. e.g. the Taito Variety Pack contains three 128 kiB games and one 64 kiB game, leaving 32 kiB unused and 32 kiB for the menu in a 512 kiB ROM chip.
+
+## Memory
+
+### 0000-3FFF - ROM Bank X0 (Read Only)
```

Comment:
The style guide says no $ on addresses

Reply:
"ROM Bank X0" (with no $) is copied off the MBC1 docs - I've checked and none of the other MBC docs use $ on the bank number in these headers either

Reply:
OK for the addresses, the context is clear enough from the "3FFF"; but I don't think it is for the "X0", so I insist on the $ there. The other docs don't have it because nobody has bothered to fix them yet—Pan Docs is full of such minor errors that have yet to be corrected.